js/components/frame.js

```diff
@@ -380,7 +380,8 @@ class Frame extends ImmutableComponent {
         case messages.LINK_HOVERED:
           method = (href, position) => {
             position = position || {}
-            let nearBottom = position.y > (window.innerHeight - 150) // todo: magic number
```

Comment:
Without this the status bar appears always left, even though the downloads bar is displayed. The value of the const is based on [this](https://github.com/brave/browser-laptop/blob/master/less/downloadBar.less#L21).